helm/frost-server/README.md

```diff
@@ -104,7 +104,7 @@ The following table lists the configurable parameters of the FROST-Server chart
 |`frost.http.ingress.path`                   | Value for the `path` of the service in the ingress spec                                                                                                                                                                                               | `/(.*)`                                                                                   |
 |`frost.http.ingress.annotations`            | Annotations for the ingress definition. E.g. to define the ingress class use ```--set 'frost.http.ingress.annotations.kubernetes\.io/ingress\.class=nginx'```                                                                                         | `nil`                                                                                     |
 |`frost.http.ingress.tls.enabled`            | Enable tls for the ingress                                                                                                                                                                                                                            | `false`                                                                                   |
-|`frost.http.ingress.tls.secretName`         | Name of the secret used for tls in the ingress definition                                                                                                                                                                                             | `nil`                                                                                     |
+|`frost.http.ingress.tls.secretName`         | Name of the secret used for tls in the ingress definition. when left blank the ingress controller will try to use it's own certificate. when using `cert-manager` and adding the right `cert-manager` annonations `cert-manager` will try to request a certificate with LetsEncrypt if desired.                                                                                                                                                                                             | `[]`                                                                                     |
```

Comment:
I'd prefer to keep the current behavior, so I think it's better that if there is no secretName, then there shouldn't be tls configured in the ingress definition.

I'm not sure if it already is implemented, like this. If yes, please adapt the description.

Reply:
Can you please explain this a bit further? You mean that if the variable `frost.http.ingress.tls.secretName` is not defined, tls ingress shouldn't be configured? so in that case, if `frost.http.ingress.tls.secretName` is defined, then specify `tls`? 

If that is the case, should we comment out  the `frost.http.ingress.tls.secretName` variable in the `values.yml` ?

Reply:
Sorry for my late review.

Sorry, I was wrong. I didn't notice that theere's the `frost.http.ingress.tls.enabled`   option. So, your solution is finde like it is :-)